install.sh

```diff
@@ -71,6 +71,7 @@ install (){
 
 # Homebrew
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+BREW_PREFIX=$(brew --prefix)
```

Comment:
Nice!
didn't know this one.
Can you apply it across the script? (nvm uses it for example)

Reply:
Done 😃 🙏